app/src/main/java/com/sopt/now/MainActivity.kt

```diff
@@ -5,11 +5,22 @@ import androidx.activity.enableEdgeToEdge
 import androidx.appcompat.app.AppCompatActivity
 import androidx.core.view.ViewCompat
 import androidx.core.view.WindowInsetsCompat
+import com.sopt.now.databinding.ActivityMainBinding
 
 class MainActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityMainBinding
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main)
+        binding = ActivityMainBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        val id = intent.getStringExtra("id")
+        val password = intent.getStringExtra("password")
+        val nickname = intent.getStringExtra("nickname")
 
+        binding.txtfieldId.text = "아이디: $id"
```

Comment:
```suggestion
        binding.txtfieldId.text = "아이디: ${intent.getStringExtra("id")}"
```
저 변수를 다른 곳에 사용하지 않는다면 이렇게 바로 넣어도 될 것 같아요!

Reply:
그리구 string으로 추출했을 시, 고정되는 값인 `"아이디:   "` 부분은 남겨두고 id만 넣을 수 있는 방법이 있는데 한번 적용해보세요!